src/format.rs

```diff
@@ -0,0 +1,81 @@
+use std::fmt;
+
+#[derive(Debug, Clone)]
+pub enum OutputFormat {
```

Comment:
It's probably better to leave this to the user. This format seems useless for us and will require extending this enum in the future.

Btw: all public enums should be annotated with `#[non_exhaustive]` (for further compatible extensions).

Reply:
On the other hand, we can use this info to better understand errors (like sent inside payload in JSON case), but probably there more seamless way